src/SchemaFactory.php

```diff
@@ -213,7 +222,8 @@ private function getDoctrineAnnotationReader(): Reader
             AnnotationRegistry::registerLoader('class_exists');
             $doctrineAnnotationReader = new DoctrineAnnotationReader();
 
-            $cache = function_exists('apcu_fetch') ? new ApcuCache() : new PhpFileCache(sys_get_temp_dir() . '/graphqlite.' . crc32(__DIR__));
+            $cacheDir = $this->annotationCacheDir ?? sys_get_temp_dir();
+            $cache = function_exists('apcu_fetch') ? new ApcuCache() : new PhpFileCache($cacheDir . '/graphqlite.' . crc32(__DIR__));
```

Comment:
We can improve the APCu check here

```suggestion
            $cache = function_exists('apcu_enabled') && apcu_enabled() ? new ApcuCache() : new PhpFileCache($cacheDir . '/graphqlite.' . crc32(__DIR__));
```